config/settings.yml

```diff
@@ -96,3 +96,59 @@ LEAFLET:
       <<: *opacity_control
     IMAGEMAPLAYER:
       <<: *opacity_control
+
+INSTITUTIONS:
```

Comment:
are we missing `Stanford` here?

Reply:
Nope.. I don't think it will ever get called based on our logic that will render this.